pybloom/pybloom.py

```diff
@@ -145,7 +147,7 @@ def _setup(self, error_rate, num_slices, bits_per_slice, capacity, count):
         self.capacity = capacity
         self.num_bits = num_slices * bits_per_slice
         self.count = count
-        self.make_hashes = make_hashfuncs(self.num_slices, self.bits_per_slice)
+        self.make_hashes, self.hashfn_name = make_hashfuncs(self.num_slices, self.bits_per_slice)
```

Comment:
Wound't it be slightly more useful to return the hash function itself and let the caller to decide how the hash function is used (get the hash function name in this case)?